tests/conftest.py

```diff
@@ -2924,3 +2924,80 @@ def finalizer():
                 collect_ocs_logs('testcases', ocp=False, status_failure=False)
 
     request.addfinalizer(finalizer)
+
+
+def get_ready_noobaa_endpoint_count(namespace):
```

Comment:
It would be better to not have functions in conftest.py. It can be in `ocs_ci/ocs/resources/mcg.py`.

Reply:
@fbalak This is not a general-purpose util function, it is there just because I hade to use the same logic in 2 diff places in the fixture.  Also, we already have some utils functions in the top of  `conftest.py`

Reply:
ok